api/v1alpha1/replicationdestination_types.go

```diff
@@ -102,6 +102,9 @@ type ReplicationDestinationRsyncSpec struct {
 	// sshUser is the username for outgoing SSH connections. Defaults to "root".
 	//+optional
 	SSHUser *string `json:"sshUser,omitempty"`
+	// paused can be used to temporarily stop replication. Defaults to "false".
+	//+optional
+	Paused bool `json:"paused,omitempty"`
```

Comment:
Do you see this as applying to methods other than rsync?
I'd expect it could apply to all (rsync, rclone, and external). If so, this should probably be defined in `ReplicationDestinationSpec`

Reply:
I'll patch this up after I record the demo